cli/asc.json

```diff
@@ -17,6 +17,18 @@
     "type": "b",
     "default": false
   },
+  "config": {
+    "category": "General",
+    "description": "Provides a configuration for assemblyscript to compile your project.",
```

Comment:
The challenge here is to keep these relatively short, i.e. not longer than any other descriptions when printed to console to avoid unnecessary wrapping or cutoff. Typically trying to keep fill words to a minimum. Here for example: "Configuration file to apply. CLI options take precedence."